Makefile

```diff
@@ -121,6 +121,11 @@ docker-push-operator:
 docker-push-registry:
 	docker push $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG)
 
+prom-rules-verify:
```

Comment:
This are kind of integration tests and belong to ./test dir I think and have to be run as part of the integration tests

Reply:
well it's technically unit test.. I think it should be part of unit tests.
it's similar to what they do on [kubevirt](https://github.com/kubevirt/kubevirt/blob/main/Makefile#L167)